cocos/ui/UIEditBox/iOS/CCUIEditBoxIOS.mm

```diff
@@ -304,6 +303,26 @@ - (void)closeKeyboard
     [self.textInput removeFromSuperview];
 }
 
+- (void)setInnerPadding:(float)left :(float)top :(float)right :(float)bottom
+{
+    if ([self.textInput isKindOfClass:[UITextField class]]) {
```

Comment:
@anjandaffo Can you please refactor the `setInnerPadding:` and `setTextAlignment` methods into the `CCUITextInput` protocol? This will get rid of all these if statements, and consolidate the code better.